client/my-sites/woocommerce/landing-page.tsx

```diff
@@ -26,10 +27,18 @@ interface Props {
 	siteId: number;
 }
 
+interface DisplayData {
+	title: string;
+	illustration: string;
+	line: string;
+	action: string;
+}
+
 const LandingPage: React.FunctionComponent< Props > = ( { siteId } ) => {
 	const { __ } = useI18n();
 	const navigationItems = [ { label: 'WooCommerce' } ];
 	const ctaRef = useRef( null );
+	const simple_seller = useIsSimpleSeller();
```

Comment:
Nitpick, but let's have this as `simpleSeller` to fit the Javascript camel-case convention.

Reply:
Fixed!